cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,255 @@
+  - The :ref:`temp directories <configuration-tmp-files>` utilized by CDAP must be writable by the default CDAP user.
+
+..
+
+- Check YARN using the YARN Resource Manager and see if the CDAP Master services are starting up.
+  Log into the cluster at ``http://<host>:8088/cluster/apps/RUNNING`` and access the YARN Resource Manager webapp. The
+  CDAP Master services should be listed under "RUNNING":
+  
+  .. image:: _images/yarn-rm-running.png
+     :align: center
+     :width: 8in
+  
+..
+
+- If CDAP Master has started, query the backend by using a command (substituting for ``<host>`` as appropriate)::
+
+    $ curl -w'\n' <host>:10000/v3/system/services/status
+    
+  The response should be something similar to::
+  
+    {"dataset.executor":"OK","metrics":"OK","transaction":"OK","appfabric":"OK","metadata.service":"OK",
+     "streams":"OK","explore.service":"OK","log.saver":"OK","metrics.processor":"OK"}
+
+..
+
+- Check that the CDAP UI can reach the CDAP Router (by default, the URL will be
+  ``http://<host>:9999`` where ``<host>`` is the IP address of one of the machines where you
+  installed the packages and started the services).
+
+
+The CDAP UI is showing a message "namespace cannot be found".
+-------------------------------------------------------------
+This is indicative that the UI cannot connect to the CDAP system service containers running in YARN.
+
+- First, check if the CDAP Master service container shows as RUNNING in the YARN ResourceManager UI.
+  The CDAP Master, once it starts, starts the other CDAP system service containers, so if it isn't running,
+  the others won't be able to start or work correctly. It can take several minutes for everything to start up.
+
+- If this doesn't resolve the issue, then it means the CDAP system services were unable to launch.
+  Ensure :ref:`YARN has enough spare memory and vcore capacity <faq-installation-startup-memory-core-requirements>`.  
+  CDAP attempts to launch between 8 and 11 containers, depending on the configuration. Check
+  the master container (Application Master) logs to see if it was able to launch all containers.
+
+- If it was able to launch all containers, then you may need to check the launched container logs for any errors.
+  The ``yarn-site.xml`` configuration file determines the container log directory.
+
+
+I don't see the CDAP Master service on YARN.
+--------------------------------------------
+- Ensure that the node where CDAP is running has a properly configured YARN client.
+  Can you log into the cluster at ``http://<host>:8088/cluster/cluster`` and access the YARN Resource Manager webapp?
+- Ensure :ref:`YARN has enough memory and vcore capacity <faq-installation-startup-memory-core-requirements>`.
+- Is the router address properly configured in the :ref:`cdap-site.xml file <configuration-options>` and the boxes using it?
```

Comment:
which router address?  this is kinda vague and not clear how it relates to "I don't see the CDAP Master service on YARN"